cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICKANSink.java

```diff
@@ -194,6 +218,20 @@ public void configure(Context context) {
                 + sslStr + ") -- Must be 'true' or 'false'");
         }  // if else
         
+        String confOrg = context.getString("pkg_by_entity", "false");
```

Comment:
In order to keep consistency on all sinks this function should be included into the DataModel scope.

For instance this PR: https://github.com/telefonicaid/fiware-cygnus/pull/1816

and the property on the agent.conf should be dataModel.

@manucarrace @AlvaroVega what do you thing about this.

Reply:
> @manucarrace @AlvaroVega @fgalan what do you thing about this?

I don't have an opinion in one sense or the other :)

I think it should be better if @manucarrace and/or @AlvaroVega provided feedback here.